### ISSUE TITLE
Improve use of Secrets Manager between Portal and Explorer

### ISSUE THREAD

**Issue by bmccandless** [chanzuckerberg/corpora-data-portal#774]
Portal and Explorer both use secrets for some configuration parameters.
In some cases the same data is located in multiple places.
For example, the auth0 client ID, is shared between portal and explorer, and between dev and staging, so it
appears in 4 secrets.
Similarly there are secrets that are shared between portal and explorer, and therefore appear in 2 secrets each.

The proposal is to 
1.  consolidate the secrets to remove duplication.

2.  make better use of cellxgene explorer's newer, and more flexible way of using secrets 
https://github.com/chanzuckerberg/cellxgene/pull/1904

3.  use terraform to set the secrets related to auth0, and switch portal and explorer to use that 
https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-infra/168

**Comment by maniarathi** [comment]
Blocked by refactoring config route.

**Comment by Bento007** [comment]
Very low priority. Closing for now.
